tests/server/TestFrameGuardConfig.js

```diff
@@ -1,16 +1,22 @@
 import { assert } from 'chai';
 import requireUncached from 'require-uncached';
+import config from 'config';
+
+const appsList = config.get('validAppNames');
 
 describe('App Specific Frameguard Config', () => {
   afterEach(() => {
     process.env.NODE_ENV = 'production';
     delete process.env.NODE_APP_INSTANCE;
   });
 
-  it('should default frameGuard to "deny"', () => {
-    const config = requireUncached('config');
-    const frameGuardConfig = config.get('frameGuard');
-    assert.equal(frameGuardConfig.action, 'deny');
-    assert.equal(frameGuardConfig.domain, undefined);
-  });
+  for (const appName of appsList) {
```

Comment:
Using `const` in a for in/of loop head like this creates a new binding for each loop iteration (like let) but the difference is the binding is immutable. In other words the `const` in the for in loop prevents re-assigning `appName` in the body.

This is also enforced by pref-const: http://eslint.org/docs/rules/prefer-const